content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -668,7 +669,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `PersistentLocalVolumes`: Enable the usage of `local` volume type in Pods.
   Pod affinity has to be specified if requesting a `local` volume.
 - `PodDisruptionBudget`: Enable the [PodDisruptionBudget](/docs/tasks/run-application/configure-pdb/) feature.
-- `PodOverhead`: Enable the [PodOverhead](/docs/concepts/scheduling-eviction/pod-overhead/)
+- `PodAffinityNamespaceSelector`: Enable the [Pod Affinity Namespace Selector](/docs/concepts/scheduling-eviction/assign-pod-node/#namespace-selector)
+  and [CrossNamespacePodAffinity](/docs/concepts/scheduling-eviction/assign-pod-node/#namespace-selector/) quota scope features.
+- `PodOverhead`: Enable the [PodOverhead](/docs/concepts/policy/resource-quotas/#cross-namespace-pod-affinity-quota)
```

Comment:
Shouldn't this be:
```suggestion
- `PodOverhead`: Enable the [PodOverhead](/docs/concepts/scheduling-eviction/pod-overhead/) feature.
```
?

Reply:
ops, this is meant for the line above, fixed.